API_CHANGELOG.md

```diff
@@ -5,8 +5,17 @@ file.
 
 This file will be updated weekly, if there are notable changes.
 
+## 2019-07-25
+* ADDED [endpoint to delete all products of a project](api_products.md#delete-produts)
+* ADDED [endpoint to delete a pricing](api_pricing.md#delete-pricing)
+* ADDED [batch endpoint for batch update
+  pricings](api_pricing.md#batch-update-pricing) and clearified
```

Comment:
ADDED hast du das batch update für Prices oder? Das für Pricings hast du "nur" extrem editiert.

Reply:
Jain, das war das was ich mit dem clearified ausdrücken wollte. Technisch gesehen ist der pricing batch der neue. Der andere heißt jetzt in der API doc nur anders